Example/Controllers/Extensions/ViewController+MiniAppUserInfoProtocol.swift

```diff
@@ -0,0 +1,18 @@
+import MiniApp
+
+extension ViewController {
```

Comment:
Shouldn't this be `extension ViewController: MiniAppUserInfoDelegate`?

Reply:
We have used Protocol composition. Since `ViewController` has already conformed to `MiniAppMessageProtocol`, we don't need the host app to conform to `MiniAppUserInfoDelegate`